src/components/celestialmap/skymap_config.js

```diff
@@ -0,0 +1,304 @@
+    interactive: false, // Enable zooming and rotation with mousewheel and dragging
+    form: false,        // Display settings form
+    // Set visiblity for each group of fields of the form
+    formFields: {
+        location: true,
+        general: true,
+        stars: true,
+        dsos: true,
+        constellations: true,
+        lines: true,
+        other: true,
+        download: false
+    },
+    controls: false,    // Display zoom controls
+    lang: "",           // Language for names, so far only for constellations: de: german, es: spanish
+                        // Default:en or empty string for english
+    container: "celestial-map",   // ID of parent element, e.g. div
+    datapath: "/data",  // Path/URL to data files, empty = subfolder 'data'
+    stars: {
+        show:true,    // Show stars
+        limit: 6,      // Show only stars brighter than limit magnitude
+        colors: true,  // Show stars in spectral colors, if not use "color"
+        style: { fill: "#ffffff", opacity: .7 }, // Default style for stars
+        designation: false,  // Show all names, including Draper and Hipparcos
+        designationType: "desig", // Which kind of name is displayed as designation (fieldname in starnames.json)
+        designationStyle: { fill: "#ddddbb", font: "10px Georgia, Times, 'Times Roman', serif", align: "left", baseline: "top" },
+        designationLimit: 4,  // Show only names for stars brighter than nameLimit
+
+        propername: false,   // Show proper name (if present)
+        propernameType: "name", // Field in starnames.json that contains proper name; may vary with culture setting
+        propernameStyle: { fill: "#ddddbb", font: "10px Georgia, Times, 'Times Roman', serif", align: "right", baseline: "bottom" },
+        propernameLimit: 3,  // Show proper names for stars brighter than propernameLimit
+
+        size: 7,       // Maximum size (radius) of star circle in pixels
+        exponent: -0.5, // Scale exponent for star size, larger = more linear
+        data: 'stars.6.json' // Data source for stellar data
+        //data: 'stars.8.json' // Alternative deeper data source for stellar data
+    },
+    dsos: {
+        show: false,    // Show Deep Space Objects
+        limit: 9,      // Show only DSOs brighter than limit magnitude
+        names: false,   // Show DSO names
+        nameLimit: 10,  // Show only names for DSOs brighter than namelimit
+        nameType: false,
+        nameStyle: { fill: "#cccccc", font: "9px Helvetica, Arial, serif", align: "left", baseline: "top" },
+        size: 6,    // Optional seperate scale size for DSOs, null = stars.size
+        exponent: 2, // Scale exponent for DSO size, larger = more non-linear
+        data: 'messier.json',  // Data source for DSOs
+        //data: 'dsos.6.json'  // Alternative broader data source for DSOs
+        //data: 'dsos.14.json' // Alternative deeper data source for DSOs
+        symbols: {  //DSO symbol styles
```

Comment:
Are these symbols actually used anywhere? It seems like in `add-custom-data.js` (and below in the styles) that you overwrite a lot of these for custom data, and it doesn't seem like any of these actually show up on the canvas.

Reply:
From reading both, I believe they are imported into add_custom_data.js, the default symbol styles for shape and fill written here are passed to the different drawing methods and depending on the actual star or deep sky object, the color and opacity can be adjusted to display more 'realistically' object color and brightness from the object's magnitude.
(Correct me if I'm wrong or misreading.)

Reply:
So actually, this part of the config deals with the default behavior for displaying stars, dsos, etc. We don't use it at the moment because their object lists are a bit unwieldy and hard to customize. And I wanted to have a working function that adds custom data, because that's the direction we're headed anyways. For pure simplicity I guess this could be removed from the config, but I left it because I find it useful to toggle various features we don't normally use during testing.